<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_page</class>
 <widget class="QWidget" name="new_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="date_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_start_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="time_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日程信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_end_Edit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="no_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yes_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
